lib/opera/operation/instructions/executors/operation.rb

```diff
@@ -16,6 +16,7 @@ def call(instruction)
             else
               result.add_errors(operation_result.errors)
               result.add_exceptions(operation_result.exceptions)
+              result.add_information(operation_result.information)
```

Comment:
also, information by design wasn't meant to hold exceptions

Reply:
we have errors for that and exceptions arrays

Reply:
https://github.com/Profinda/opera/commit/e5e83ca1b3fce96e4e85e2d100327f038114d131#diff-828218ef941ed5d36e1ab826614301e149b77cbc896665aaae4b327d3bee4449R17

Reply:
It's not for storing exceptions or errors, that's right, but we should keep the information that is collected on the nested operations as that's useful. I'll add it to the success path as well.

Reply:
It might be misleading and not readable. We are adding information stright to parent operations. Imagine situation that you have 5 level deep structure and the same information on every level: `Executed in 5s`. How can you know from which operations it came? If we want to do so (I am not sure about that), we should do a similar thing that we did for `execution`

Reply:
not sure I agree with this. With current approach we have flexibility to either add to parent or add to child as 
```
add_information(child: { status: 404 })
```

or 

```
add_information(status: 404)
```

In many cases you don't want to case about the actual source of 404 state, but you want to add it.